### PR TITLE
Accessibility Fixes

### PR DESCRIPTION
**Pull request by chadbrokaw-exygy**
[DAH-1395](https://sfgovdt.jira.com/browse/DAH-1395)

[DAH-1395]: https://sfgovdt.jira.com/browse/DAH-1395?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/javascript/layouts/Layout.tsx [hunk 1]
```diff
@@ -170,8 +170,12 @@ const Layout = (props: LayoutProps) => {
           imageOnly={true}
           mobileText={true}
           logoWidth={"medium"}
+          logoClass="translate"
           menuLinks={getMenuLinks(!!profile, signOut)}
-          strings={{ skipToMainContent: t("t.skipToMainContent") }}
+          strings={{
+            skipToMainContent: t("t.skipToMainContent"),
+            logoAriaLable: t("t.dahliaSanFranciscoHousingPortal"),
```
Comment (christine-sfg) [inline app/javascript/layouts/Layout.tsx:177]: this one we may need to add to the [glossary](https://sfgovdt.jira.com/wiki/spaces/HOUS/pages/3189538929/Updating+DAHLIA+translations+using+Phrase+and+Freeway#SF.gov-Glossary)

File: app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx [hunk 2]
```diff
@@ -50,17 +50,20 @@ export const ListingDetailsImageCard = ({ listing }: ListingDetailsImageCardProp
...
-            onFocus={() => setShareImage(shareButtonSelected)}
-            onMouseEnter={() => setShareImage(shareButtonSelected)}
-            onMouseLeave={() => setShareImage(shareButton)}
-            target="_blank"
-            className="share-button"
-          >
-            <img alt={t("label.share")} src={shareImage} />
-          </a>
+          <div>
+            <a
```
Comment (kramduckner) [inline app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx:54]: I might just being blind! But I can't find this in the Review Instructions.

**Review by kramduckner — COMMENTED** [review]
Looking awesome! Just a couple questions.

File: app/javascript/__tests__/modules/listingDetailsLottery/__snapshots__/ListingDetailsLotterySearchForm.test.tsx.snap [hunk 3]
```diff
@@ -45,7 +45,9 @@ exports[`ListingDetailsLotteryModal displays initial view with form and listing
       </div>
     </div>
     <button
-      className="bg-blue-500 h-12 mt-2 p-3"
```
[the remark this answers is not in the document]
Comment (chadbrokaw-exygy) [reply app/javascript/__tests__/modules/listingDetailsLottery/__snapshots__/ListingDetailsLotterySearchForm.test.tsx.snap:48]: I have no idea why that changed, but I changed it back!

File: app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx (hunk 2 above)
Comment (chadbrokaw-exygy) [reply app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx:54]: Ugh, I borked a rebase. We need to add aria labels to the link and alt text to the image, but adding this div should not be a part of it. Thank you for catching!

Comment (kramduckner) [reply app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx]: ha no worries! I totally understand

**Review by kramduckner — APPROVED** [review]
lgtm!

**Pull request merged**